### PR TITLE
add image command for crictl

### PR DESCRIPTION
**Pull request by heartlock**
Signed-off-by: heartlock <21521209@zju.edu.cn>
ref:  #36 
@feiskyer @xlgao-zju

### CODE REVIEW

File: cmd/crictl/image.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+var pullImageCommand = cli.Command{
+	Name:  "pull",
+	Usage: "pull an image",
+	Action: func(context *cli.Context) error {
+		_, err := PullImage(imageClient, context.Args().Get(0))
+		if err != nil {
+			return fmt.Errorf("pulling image failed: %v", err)
+		}
+		return nil
```
Comment (feiskyer) [inline cmd/crictl/image.go]: Please also print out the pulled image_ref when success

File: cmd/crictl/image.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+			fmt.Printf("ID: %s\n", image.Id)
+			for _, tag := range image.RepoTags {
+				fmt.Printf("Tag: %s\n", tag)
+			}
+			for _, digest := range image.RepoDigests {
+				fmt.Printf("Digest: %s\n", digest)
+			}
+			if image.Size_ != 0 {
+				fmt.Printf("Size: %d\n\n", image.Size_)
+			}
```
Comment (feiskyer) [inline cmd/crictl/image.go]: missing uid and username?

**Comment by heartlock** [comment]
updated it! @feiskyer

**Comment by feiskyer** [comment]
@heartlock LGTM. Thanks.

**Pull request merged**